src/net.cpp

```diff
@@ -222,10 +221,12 @@ void AdvertiseLocal(CNode *pnode)
         }
         if (addrLocal.IsRoutable() || gArgs.GetBoolArg("-addrmantest", false))
         {
-            LogPrint(BCLog::NET, "AdvertiseLocal: advertising address %s\n", addrLocal.ToString());
-            pnode->PushAddress(addrLocal, rng);
+            LogPrint(BCLog::NET, "GetLocalAddrForPeer: advertising address %s to peer=%d\n", addrLocal.ToString(), pnode->GetId());
```

Comment:
in the first commit: Would be good to remove the function name when touching this. You can use #19809 to get the function name in the log message.

Reply:
Done